birdhouse/optional-components/prometheus-longterm-metrics/default.env

```diff
@@ -0,0 +1,30 @@
+export PROMETHEUS_LONGTERM_VERSION='${PROMETHEUS_VERSION:-"v2.52.0"}'
+export PROMETHEUS_LONGTERM_DOCKER='${PROMETHEUS_DOCKER:-prom/prometheus}'
```

Comment:
Should probably consider using `PROMETHEUS_VERSION` and `PROMETHEUS_DOCKER` as defaults instead of the hardcoded values. Otherwise, we must remember to update both combinations of variables each time.

Reply:
This will use those variables as defaults and only fall back to the hardcoded versions if those variables are not set.
This is necessary to allow for the use case where the longterm metrics component is deployed on a different server/stack from the prometheus component (as requested by @tlvu here: https://github.com/bird-house/birdhouse-deploy/pull/461#issuecomment-2176793022).

Reply:
Ok 👍